packages/components/tag/stories/Tag.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
-import Tag, { TagProps } from './Tag';
-import SectionHeading from '../Typography/SectionHeading';
-import Flex from '../Flex/Flex';
+import { Tag, TagProps } from '../src/Tag';
+import SectionHeading from '../../../forma-36-react-components/src/components/Typography/SectionHeading/SectionHeading';
+import Flex from '../../../forma-36-react-components/src/components/Flex/Flex';
```

Comment:
later we would be able to use `import { SectionHeading } from '@contentful/f36-typography'` correct?

Reply:
yep, exactly